open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-server/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/auditlog/VirtualizationAuditCode.java

```diff
@@ -0,0 +1,130 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.governanceservers.virtualizationservices.auditlog;
+
+import org.odpi.openmetadata.repositoryservices.auditlog.OMRSAuditLogRecordSeverity;
+
+import java.text.MessageFormat;
+
+public enum VirtualizationAuditCode {
+
+    SERVICE_INITIALIZING("VIRTUALIZER-0001",
+            OMRSAuditLogRecordSeverity.INFO,
+            "The Virtualizer is initializing a new server instance",
+            "The local server has started up a new instance of the Virutalizer.",
+            "No action is required.  This is part of the normal operation of the service."),
+    SERVICE_INITIALIZED("VIRTUALIZER-0002",
+            OMRSAuditLogRecordSeverity.INFO,
+            "The Virtualizer has initialized a new instance for server {0}",
+            "The local server has completed initialization of a new instance.",
+            "No action is required.  This is part of the normal operation of the service."),
+    SERVICE_SHUTDOWN("VIRTUALIZER-0003",
+            OMRSAuditLogRecordSeverity.INFO,
+            "The Virtualizer is shutting down its instance for server {0}",
+            "The local server has requested shut down of a Virtualizer instance.",
+            "No action is required.  This is part of the normal operation of the service."),
+    IV_OUT_TOPIC_CONNECTOR_INITIALIZED("VIRTUALIZER-0004",
+            OMRSAuditLogRecordSeverity.INFO,
+            "The Virtualizer has initialized a Kafka connector for Information View Out Topic",
```

Comment:
Should not refer to Kafka since the event bus may not be kafka.

Reply:
fixed